MinecraftClient/McClient.cs

```diff
@@ -216,6 +216,7 @@ private void StartClient(string user, string uuid, string sessionID, string serv
 
                     //Add your ChatBot here by uncommenting and adapting
                     //BotLoad(new ChatBots.YourBot());
+                    BotLoad(new MineCube());

```

Comment:
Remember to remove this line after you have done your work.

Reply:
I think I will make this a script in the config folder later. It is not necessary to implement a full bot to the client.